### PR TITLE
Update elastic cloud sign up link with trackable url

### PR DESCRIPTION
**Pull request by saarikabhasi**
* Update existing Elastic cloud sign up url with `utm_source` and `utm_content`
* Add trackable sign up link if signup link was not there 
* Update Elasticsearch-labs openai cookbook with trackable url `utm_content=openai-cookbook`

### CODE REVIEW

File: notebooks/search/03-ELSER.ipynb [hunk 1]
```diff
@@ -39,7 +41,7 @@
    "source": [
     "# Create Elastic Cloud deployment\n",
     "\n",
-    "If you don't have an Elastic Cloud deployment, sign up [here](https://cloud.elastic.co/registration?fromURI=%2Fhome) for a free trial.\n",
+    "If you don't have an Elastic Cloud deployment, sign up [here](https://cloud.elastic.co/registration?utm_source=github&utm_content=elasticsearch-labs-notebook?fromURI=%2Fhome) for a free trial.\n",
```
Comment (joemcelroy) [inline notebooks/search/03-ELSER.ipynb:44]: two question marks. remove the fromURI param.

File: notebooks/search/04-multilingual.ipynb [hunk 2]
```diff
@@ -37,7 +37,7 @@
     "\n",
     "- Python 3.6 or later\n",
     "- An Elastic deployment with minimum **4GB machine learning node**\n",
-    "   - We'll be using [Elastic Cloud](https://www.elastic.co/guide/en/cloud/current/ec-getting-started.html) for this example (available with a [free trial](https://cloud.elastic.co/registration?elektra=en-ess-sign-up-page))\n",
+    "   - We'll be using [Elastic Cloud](https://www.elastic.co/guide/en/cloud/current/ec-getting-started.html) for this example (available with a [free trial](https://cloud.elastic.co/registration?utm_source=github&utm_content=elasticsearch-labs-notebook?elektra=en-ess-sign-up-page))\n",
```
Comment (joemcelroy) [inline notebooks/search/04-multilingual.ipynb:40]: remove `?elektra=en-ess-sign-up-page`

File: notebooks/search/03-ELSER.ipynb (hunk 1 above)
Comment (saarikabhasi) [reply notebooks/search/03-ELSER.ipynb:44]: I might have added `fromURI` param in multiple instance. Will update

Comment (saarikabhasi) [reply notebooks/search/03-ELSER.ipynb]: added in 5725f74106dc37b600e6ba7f4f4433ad20e7bb7c

File: notebooks/search/04-multilingual.ipynb (hunk 2 above)
Comment (saarikabhasi) [reply notebooks/search/04-multilingual.ipynb]: added in 5725f74106dc37b600e6ba7f4f4433ad20e7bb7c

**Review by joemcelroy — APPROVED** [review]

**Pull request merged**